tests/installation/partitioning_splitusr.pm

```diff
@@ -27,12 +27,21 @@ sub run {
 
     # select root partition
     send_key_until_needlematch 'vda-selected', 'right';
+    wait_screen_change { send_key 'alt-p' if is_storage_ng };
```

Comment:
If it's not storage_ng you wait for screen change without action:  `wait_screen_change {}` - does it work?

Reply:
exactly, execute only in storage-ng systems